src/pages/platform/[id]/observations/[type].tsx

```diff
@@ -0,0 +1,70 @@
+import { GetStaticPaths, GetStaticProps } from "next"
+import { useRouter } from "next/router"
+import React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6288046)